.github/workflows/build_daily.yaml

```diff
@@ -3,6 +3,7 @@
 # Triggers:
 #   1. Daily trigger at 04:00 (Beijing, UTC+08:00)
 #   2. Manual trigger
+#   3. Push tag with prefix "release"
```

Comment:
How about changing the name to build_release.yaml?

Reply:
This workflow has nothing to do with release，but only auto triggerred when a release tag is pushed. 
Based on the conclusions of the previous meeting，the name is build_daily.